lib/writer_buffered.c

```diff
@@ -141,7 +141,12 @@ BufferedWriterParam(BufferedWriter *self, const char *keyword, char *value)
 		ASSERT_ONCE(self->base.output == NULL);
 
 		self->base.relid = RangeVarGetRelid(makeRangeVarFromNameList(
-						stringToQualifiedNameList(value)), NoLock, false);
```

Comment:
Can we remove the line because stringToQualifiedNameList() is define in lib/pgut/pgut-be.h?